cmd/atlas/apply.go

```diff
@@ -29,3 +39,27 @@ func init() {
 	cobra.CheckErr(applyCmd.MarkFlagRequired("dsn"))
 	cobra.CheckErr(applyCmd.MarkFlagRequired("file"))
 }
+
+func applyRun(d *Driver, u schemaUnmarshaler, dsn string, file string) {
+	ctx := context.Background()
+	name, err := schemaNameFromDSN(dsn)
+	cobra.CheckErr(err)
+	s, err := d.Inspector.InspectSchema(ctx, name, nil)
+	cobra.CheckErr(err)
+	f, err := ioutil.ReadFile(file)
+	cobra.CheckErr(err)
+	var desired schema.Schema
+	err = u.unmarshal(f, &desired)
+	changes, err := d.Differ.SchemaDiff(s, &desired)
+	cobra.CheckErr(err)
+	prompt := promptui.Select{
+		Label: "Are you sure?",
+		Items: []string{"Continue", "Abort"},
+	}
+	_, result, err := prompt.Run()
+	cobra.CheckErr(err)
+	if result == "Continnue" {
+		err = d.Execer.Exec(ctx, changes)
```

Comment:
d.Exec

Reply:
OK, removed the DB the driver has since we don't need it at the moment.